modules/spring-config/src/main/resources/META-INF/mule.xsd

```diff
@@ -4228,6 +4190,13 @@
                         </xsd:documentation>
                     </xsd:annotation>
                 </xsd:attribute>
+                <xsd:attribute name="ignoreErrorType" type="errorIdentifierType" use="optional">
```

Comment:
not `filterOnErrorType`?  Isn't it filtering it from resulting collection? Or did you determine that it's read-only with respect to the original collection in the end?